releases.go

```diff
@@ -122,6 +127,7 @@ func (s *releasesService) Rollback(ctx context.Context, db *gorm.DB, opts Rollba
 		App:         app,
 		Config:      r.Config,
 		Slug:        r.Slug,
+		Formation:   r.Formation,
```

Comment:
One of the really cool things about this change is that the rollback operation is significantly simpler. Since we now have a Formation object, we can just copy the old formation to the new release.